doc/devel/documenting_mpl.rst

```diff
@@ -213,38 +220,52 @@ is better than:
 In addition, since underscores are widely used by Sphinx itself, use
 hyphens to separate words.
 
+.. _referring-to-other-code:
 
 Referring to other code
 -----------------------
 
 To link to other methods, classes, or modules in Matplotlib you can use
 back ticks, for example:
 
-.. code-block:: python
+.. code-block:: rst
 
-  `~matplotlib.collections.LineCollection`
+  `matplotlib.collections.LineCollection`
 
-returns a link to the documentation of
-`~matplotlib.collections.LineCollection`.  For the full path of the
-class to be shown, omit the tilde:
+generates a link like this: `matplotlib.collections.LineCollection`.
 
-.. code-block:: python
+*Note:* We use the sphinx setting `default_role = 'obj'` so that you don't
+have to use qualifiers like `:class:`, `:func:`, `:meth:` and the likes.
```

Comment:
On that note, I believe that these need to be double-backticks in rST, no?

Reply:
I thought this as well, but I believe we are using the very nice option that makes the documentation build fail if there's any warnings, so I guess these are correct.

Reply:
it may not give a warning though, it may just not display as in-line code, but as regular text w/ backticks

Reply:
You're right. These should be double-backticks. Fixed.